src/actions/absoluteAction.ts

```diff
@@ -0,0 +1,47 @@
+module Plywood {
+  export class AbsoluteAction extends Action {
+    static fromJS(parameters: ActionJS): AbsoluteAction {
+      return new AbsoluteAction(Action.jsToValue(parameters));
+    }
+
+    constructor(parameters: ActionValue) {
+      super(parameters, dummyObject);
+      this._ensureAction("absolute");
+    }
+
+    public getOutputType(inputType: string): string {
+      this._checkInputType(inputType, 'NUMBER');
+      return 'NUMBER';
+    }
+
+    protected _getFnHelper(inputFn: ComputeFn): ComputeFn {
+      return (d: Datum, c: Datum) => {
+        var v = Math.abs(inputFn(d, c));
```

Comment:
This should 'forward' `null` see how concat works for example. Also `v` is redundant